feat(@refinedev/cli): add "--platform" option to specify project type (#4833)
refinedev/refine: .changeset/shiny-cooks-push.md

---
"@refinedev/cli": minor
---

> add "--platform" option to run, dev, and build to specify which project type to use for the commands

Reviewer: ```suggestion
add "--platform" option to start, dev, and build to specify project type externally
```
